LSTM/TimeSeries-Multivariate/src/LSTMTimeSeriesMultivariate.cpp

```diff
@@ -43,37 +61,114 @@ double MSE(arma::cube& pred, arma::cube& Y)
 }
 
 /*
- * The time series data for today should contain the [Volume of stocks traded,
- * Opening stock price, Closing stock price, Min stock price, Max stock price]
- * for past 'rho' days and the target variable will be Google’s
- * stock price today (high, low) and so on.
+ * The time series data for training the model contains the Closing stock price, the Volume of stocks traded,
+ * Opening stock price, Highest stock price and Lowest stock price for 'rho' days in the past. 
+ * The two target variables (multivariate) we want to predict are the Highest stock price and Lowest stock price
+ * (high, low) for the next day! 
+ *
+ * NOTE: Please note that we do not use the last input data point in the training because there is no target 
+ *       (next day (high, low)) for that point!
  */
 template<typename InputDataType = arma::mat,
-     typename DataType = arma::cube,
-     typename LabelType = arma::cube>
-void CreateTimeSeriesData(InputDataType dataset, DataType& X, LabelType& y, size_t rho)
+  typename DataType = arma::cube,
+  typename LabelType = arma::cube>
+  void CreateTimeSeriesData(InputDataType dataset, DataType& X, LabelType& y, size_t rho)
```

Comment:
```suggestion
void CreateTimeSeriesData(InputDataType dataset, DataType& X, LabelType& y, size_t rho)
```
Seems like no need for extra spacing here. :+1: